src/main/java/at/ac/fhcampuswien/fhmdb/ui/MovieCell.java

```diff
@@ -9,35 +10,45 @@
 
 public class MovieCell extends ListCell<Movie> {
 
+    private final ClickEventHandler<Movie> clickEventHandler;
+
```

Comment:
### Remove Button not shown in the Movie-Cell

Add this here:
`private final boolean isWatchlistCell;`